src/Compilers/Core/Portable/Symbols/IMethodSymbol.cs

```diff
@@ -135,6 +141,8 @@ public interface IMethodSymbol : ISymbol
         /// </summary>
         ITypeSymbol ReceiverType { get; }
 
+        Nullability ReceiverTypeNullability { get; }
```

Comment:
A receiver can never be null (except in a reduced extension method). I suggest just removing this API.